packages/sanes-chrome-extension/src/routes/account/index.dom.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { TokenTicker } from '@iov/bcp';
+import TestUtils from 'react-dom/test-utils';
+import { Store } from 'redux';
+import * as messages from '../../extension/background/messages';
+import { CreatePersonaResponse, GetPersonaResponse } from '../../extension/background/messages';
+import { PersonaAcccount, ProcessedTx } from '../../logic/persona';
+import { aNewStore } from '../../store';
+import { RootState } from '../../store/reducers';
+import { whenOnNavigatedToRoute } from '../../utils/test/navigation';
+import { withChainsDescribe } from '../../utils/test/testExecutor';
+import { ACCOUNT_STATUS_ROUTE } from '../paths';
+import { processSignUp } from '../signup/test/travelToSignup';
+import { travelToAccount } from './test/travelToAccount';
+
+withChainsDescribe('DOM > Feature > Account Status', () => {
+  const accountMock: PersonaAcccount = { label: 'Account 0' };
+  const mnemonic = 'badge cattle stool execute involve main mirror envelope brave scrap involve simple';
+  const txMock: ProcessedTx = {
+    id: '111',
+    recipient: 'Example Recipient',
+    signer: 'Example Signer',
+    amount: { quantity: '10', fractionalDigits: 3, tokenTicker: 'ETH' as TokenTicker },
+    time: 'Sat May 25 10:10:00 2019 +0200',
+    error: null,
+  };
+
+  const createPersonaResponse: CreatePersonaResponse = {
+    accounts: [accountMock],
+    mnemonic,
+    txs: [txMock],
+  };
+
+  const getPersonaResponse: GetPersonaResponse = {
+    accounts: [accountMock],
+    mnemonic,
+    txs: [txMock],
+  };
+
+  let store: Store<RootState>;
+  let accountStatusDom: React.Component;
+
+  beforeEach(async () => {
+    store = aNewStore();
+
+    jest.spyOn(messages, 'sendCreatePersonaMessage').mockResolvedValue(createPersonaResponse);
+
+    await processSignUp(store);
+    await whenOnNavigatedToRoute(store, ACCOUNT_STATUS_ROUTE);
```

Comment:
I think you do not need to check for `await whenOnNavigatedToRoute(store, ACCOUNT_STATUS_ROUTE);` , because this is redundant in this test and not required, you are not testing sign up process in this test case.